pom.xml

```diff
@@ -945,6 +945,7 @@ under the License.
     <javaVersion>8</javaVersion>
     <maven.compiler.source>${javaVersion}</maven.compiler.source>
     <maven.compiler.target>${javaVersion}</maven.compiler.target>
+    <maven.compiler.proc>none</maven.compiler.proc>
```

Comment:
maybe a small comment would help here, saying that this needs to be overwritten with empty value to restore default annotation processing behaviour.